### DISCUSSION TITLE
Require user to confirm password in second box before creating account or resetting password

### DISCUSSION THREAD

**Discussion by shipsides-sparknel** [directus/directus#15469] [Feature Requests]
Eliminate potential user error by having the create account/reset password page a second password box to fill out, to confirm that the user has entered the correct password twice before creating it.

Simple but effective!

**Comment by rijkvanzanten** [comment]
Can't wait for the FIDO standard to be ubiquitous 😄

**Reply by rijkvanzanten** [reply]
(For those interested/confused: https://www.theverge.com/2022/5/5/23057646/apple-google-microsoft-passwordless-sign-in-fido)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
